spec/classes/nginx_spec.rb

```diff
@@ -190,7 +190,7 @@
 
             it { is_expected.to contain_package('nginx') }
 
-            if (facts.dig(:os, 'name') == 'Debian' && %w[10 11].include?(facts.dig(:os, 'release', 'major'))) ||
+            if (facts.dig(:os, 'name') == 'Debian' && %w[11].include?(facts.dig(:os, 'release', 'major'))) ||
                (facts.dig(:os, 'name') == 'Ubuntu' && %w[bionic focal jammy].include?(facts.dig(:os, 'distro', 'codename')))
```

Comment:
These all look like modern versions. Is there anything that still uses plain `passenger` or  can it be simplified to just `facts.dig(:os, 'family') == 'Debian'`?